docs/DEVGUIDE.md

```diff
@@ -1,10 +1,15 @@
-# An End-to-End Walkthrough
+# Developer's Guide to Service-Catalog
 
-## Prerequisites and Setup
-
-You will need to meet the following prerequisites:
+## Prerequisites
 
+At a minimum you will need:
 * [Docker](https://www.docker.com) installed locally
+* GNU Make
+* [git](https://git-scm.com)
+
+These will allow you to build and test using Docker.
+
+If you want to build without Docker, then you will also need:
```

Comment:
Now that we're differentiating between the prereqs for building in-container vs out-of-container (which I like a lot, btw), I think there are some things in the out-of-container list (below) that don't belong...

For instance, why is it a requirement for building out-of-container that one has a working kubernetes cluster? It seems to me this is actually not a build requirement, but rather a _deployment_ requirement and that whether the binaries were built in-container or out-of-container is inconsequential.

Maybe we need a third section that enumerates _deployment_ requirements separately from the in-container or out-of-container _build_ requirements?